deploy/roles/r-libs/tasks/main.yml

```diff
@@ -0,0 +1,111 @@
+---
+- name: Install packages required by R packages
+  become: yes
+  become_method: sudo
+  yum:
+    name: "{{ item }}"
+    state: present
+    disable_gpg_check: yes
+  with_items: "{{ r_dependencies }}"
+  tags:
+    - r-update
+  when: install_r_updates
+
+- name: Find any locked directories
+  become: yes
+  become_method: sudo
+  find:
+    paths: "{{ r_system_package_dir }}"
+    pattern: '00LOCK-*'
+    file_type: directory
+  register: locked_dirs
+  tags:
+    - r-update
+  when: install_r_updates
+
+- name: Remove locked directories
+  become: yes
+  become_method: sudo
+  file:
+    path: "{{ item.path }}"
+    state: absent
+  with_items:
+    - "{{ locked_dirs.files }}"
+  tags:
+    - r-update
+
+- include_role:
```

Comment:
@lel99999 We don't want to include the role in here, that is already covered by the playbooks and dependencies.